sqls/events/0x02d5c618dbc591544b19d0bf13543c0728a3c4ec/Aggregator_event_AnswerUpdated.json

```diff
@@ -0,0 +1,51 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "name": "current",
+                    "type": "int256"
+                },
+                {
+                    "indexed": true,
+                    "name": "roundId",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "name": "timestamp",
+                    "type": "uint256"
+                }
+            ],
+            "name": "AnswerUpdated",
+            "type": "event"
+        },
+        "contract_address": "0x02d5c618dbc591544b19d0bf13543c0728a3c4ec",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "Aggregator",
```

Comment:
should be `chainlink` for all files

Reply:
For the dataset_name?

Reply:
> For the dataset_name?
Yup, the "dataset_name" for all the files should be "chainlink"

Reply:
done